keyboards/4by3/info.json

```diff
@@ -0,0 +1,35 @@
+{
+    "keyboard_name": "4by3",
+    "url": "https://github.com/eliassjogreen/4by3",
+    "maintainer": "eliassjogreen",
+    "width": 4,
+    "height": 3,
+    "layouts": {
+        "LAYOUT_horizontal": {
+            "key_count": 12,
+            "layout": [
+                { "x": 0, "y": 0 }, { "x": 1, "y": 0 }, { "x": 2, "y": 0 }, { "x": 3, "y": 0 },
+                { "x": 0, "y": 1 }, { "x": 1, "y": 1 }, { "x": 2, "y": 1 }, { "x": 3, "y": 1 },
+                { "x": 0, "y": 2 }, { "x": 1, "y": 2 }, { "x": 2, "y": 2 }, { "x": 3, "y": 2 }
+            ]
+        },
+        "LAYOUT_vertical_right": {
+            "key_count": 12,
+            "layout": [
+                { "x": 0, "y": 2 }, { "x": 0, "y": 1 }, { "x": 0, "y": 0 },
```

Comment:
Lines 19-22 should actually be:

```json
                { "x": 0, "y": 0 }, { "x": 1, "y": 0 }, { "x": 2, "y": 0 },
                { "x": 0, "y": 1 }, { "x": 1, "y": 1 }, { "x": 2, "y": 1 },
                { "x": 0, "y": 2 }, { "x": 1, "y": 2 }, { "x": 2, "y": 2 },
                { "x": 0, "y": 3 }, { "x": 1, "y": 3 }, { "x": 2, "y": 3 }
```

The layout macro takes care of mapping the physical location to the electrical position, but it doesn't provide any information about key sizes or positions, which is what this does.

Your current implementation would force QMK Configurator users to map the keys as follows to get the `arrowpad` keymap:

![image](https://user-images.githubusercontent.com/18669334/63217913-93307580-c103-11e9-88b7-dbcaea4ede7c.png)

... which can be misleading.